config/prow_common.mk

```diff
@@ -121,3 +121,25 @@ update-testgrid-config: confirm-master
 		"--output=gs://$(TESTGRID_GCS)/config" \
 		"--yaml=$(realpath $(TESTGRID_CONFIG))"
 
+.PHONY: verify-testgrid-config
```

Comment:
I was considering adding this check as a presubmit test job, similar as how we added the `checkconfig` job.
Or do you prefer adding it as a make target?

Reply:
Ah. I see, how do you run 2 commands in a presubmit?